app/assets/javascripts/application/popover.js

```diff
@@ -0,0 +1,16 @@
+mumuki.load(() => {
+  let $popover = $('.mu-popover');
+
+  $('html').click(function(e) {
```

Comment:
This causes that even if you click _within_ the popover, it still closes, which seems odd.

Reply:
Done